src/client/components/LastMessageList/LastMessagesList.js

```diff
@@ -5,24 +5,27 @@ import './LastMessagesList.css';
 import { object } from '@storybook/addon-knobs';
 
 export default function LastMessageList({ lastChannels }) {
-  if (!lastChannels) {
+  if (lastChannels === null) {
```

Comment:
The previous check was better since it checked for both `null` and `undefined`.

Reply:
Changed